docs/integrations_api.yml

```diff
@@ -0,0 +1,134 @@
+          type: string
+      responses:
+        200:
+          description: |
+            Authentication successful - a new JWT is issued and returned.
+            The JWT is signed with the API's private key ('RS256' signing algorithm),
+            and contains the following standard claims:
+            * 'iss' - issuer
+            * 'exp' - expiry date
+            * 'sub' - username
+            * 'jti' - token's unique identifier (tracked for the purpose of revocation)
+
+            A custom 'scp' (scopes) claim is added to distinguish between a regular token and a first-time user
+            creation token:
+                * 'scp' : ['mender.*'] - regular token
+                * 'scp' : ['mender.users.create.initial'] - first-time user creation token
+          examples:
+            application/jwt: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.
+                             eyJleHAiOjE0NzYxMTkxMzYsImp0aSI6Ijg1NGIzMTA5LTQ4NjItNGEyNS1hMWZ
+                             iLWYxMTE2MWNlN2E4NCIsImlzcyI6Ik1lbmRlciIsInN1YiI6InVzZXJAYWNtZS
+                             5jb20iLCJzY3AiOlsibWVuZGVyLioiXX0.
+                             XPKNdZA_n5c7I0sNPSz-Jhm6ydteO5QSifzoaqevyc4
+
+        400:
+          description: Bad request, see error message for details.
+          schema:
+            $ref: '#/definitions/Error'
+        401:
+          description: Unauthorized.
+          schema:
+            $ref: '#/definitions/Error'
+        500:
+          description: Internal server error.
+          schema:
+            $ref: '#/definitions/Error'
+
+  /users/inital:
+    post:
+      summary: Create the initial Mender user
+      description: |
+        Creates the initial Mender user, provided that:
+        * the users database is empty
+        * a valid JWT token is presented (scope: 'mender.users.create.initial')
+      parameters:
+        - name: Authorization
+          in: header
+          required: true
+          type: string
+          format: Bearer [token]
+          description: Contains the JWT token issued upon login.
+        - name: user
+          in: body
+          description: New user descriptor.
+          required: true
+          schema:
+            $ref: "#/definitions/User"
+      produces:
+        - application/json
+      responses:
+        201:
+          description: Successful response.
+          headers:
+            Location:
+              description: URL of the newly created user.
+              type: string
+        400:
+          description: Bad request, see error message for details.
+          schema:
+            $ref: '#/definitions/Error'
+        401:
+          description: Unauthorized.
+          schema:
+            $ref: '#/definitions/Error'
+        500:
+          description: Internal server error.
+          schema:
+            $ref: '#/definitions/Error'
+
+definitions:
+  User:
+    description: New user descriptor.
+    type: object
+    properties:
+      name:
```

Comment:
ok, I guess Eystein wants editable emails :/ I'll have to change the `sub` claim in JWT to use the internal IDs.